samples/nrf9160/lte_ble_gateway/README.rst

```diff
@@ -44,8 +44,6 @@ Two buttons and two switches are used to enter a pairing pattern to associate a
 
 When the connection is established, set switch 2 to **N.C.** to send simulated GNSS data to nRF Cloud once every 2 seconds.
 
-See the :ref:`asset_tracker_user_interface` in the :ref:`asset_tracker` documentation for detailed information about the different LED states used by the sample.
```

Comment:
@nordic-hani FYI, the gateway sample docs should probably be updated when this reference is removed